library/src/morpheus-master/sass/modules/joinpage/_base.scss

```diff
@@ -0,0 +1,45 @@
+.#{$namespace}joinpage {
+    position: fixed;
+    width: 100%;
+    min-height: 14em;
+    background: $body-background-color;
+
+    .join {
```

Comment:
This sizing and positioning isn't the best way to do it for responsive design, but it matches the xlogin page and replaces the former table layout. It will work for now and will be improved with SWITCH later on.